clients/src/test/java/org/apache/kafka/clients/ClientUtilsTest.java

```diff
@@ -51,10 +52,12 @@ public void testParseAndValidateAddressesWithReverseLookup() {
         checkWithoutLookup("[::1]:8000");
         checkWithoutLookup("[2001:db8:85a3:8d3:1319:8a2e:370:7348]:1234", "localhost:10000");
         List<InetSocketAddress> validatedAddresses = checkWithLookup(Arrays.asList("example.com:10000"));
-        assertEquals(2, validatedAddresses.size());
-        InetSocketAddress address = validatedAddresses.get(0);
-        assertEquals("93.184.216.34", address.getHostName());
-        assertEquals(10000, address.getPort());
+        assertTrue("Unexpected addresses " + validatedAddresses, validatedAddresses.size() >= 1);
```

Comment:
Can we add a comment explaining that the expected result is 1 or 2 depending on whether both ipv4 and ipv6 are enabled or not?

Reply:
@ijuma Thanks for the review, added comment